akka-projection-slick/src/test/scala/akka/projection/slick/SlickOffsetStoreSpec.scala

```diff
@@ -39,7 +39,7 @@ object SlickOffsetStoreSpec {
 
       # TODO: configure connection pool and slick async executor
       db = {
-       url = "jdbc:h2:mem:offset-store-test-slick;DB_CLOSE_DELAY=-1"
+       url = "jdbc:h2:mem:slick-offset-store-test-slick;DB_CLOSE_DELAY=-1"
```

Comment:
that name has 2x slick

Reply:
the jdbc one was also duplicated, I need more coffee

Reply:
It also reveals the reason why the 'desperate' on the title. I'm just trying out stuff and hitting my keyboard randomly.